### PR TITLE
Add documentation regarding large file handling

### PR DESCRIPTION
**Pull request by drewbo**
Proposed changes in this pull request:
- Add documentation regarding large file handling

/cc @apburnes @hursey013 @Ephraim-G 

[:sunglasses: PREVIEW](https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/federalist.18f.gov/ddb-large-file-handling/)

### CODE REVIEW

**Review by apburnes — COMMENTED** [review]
This looks like a good start. Maybe we add info about additional strategies for saving space and accommodating more data like:

-  Creating addition federalist sites to specifically manage media or file types. (`media.example.gov`, `reports.example.gov`)
- Compressing files meant to be downloaded and optimizing images
- Not declaring certain node.js development dependencies directly in their `package.json` and using `npx <dependency script>`

**Comment by drewbo** [comment]
@apburnes
- for the asset sites, I added the frtib.gov example, should I abstract that and explain more?
- do we have a recommended tool (or link to an template/example) for compression?
- 👍

File: pages/documentation/large-file-handling.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Regular build container provide 2GB of total user space. Because dependencies like those included in your `Gemfile` or `package.json` will be installed here, it's a good rule of thumb to keep your total Github repository size under 1GB.
+- If your site starts hitting this initial limit, you can request an upgrade to 4GB of user space. Send an email to [federalist-support@gsa.gov](mailto:federalist-support@gsa.gov).
+- If your site is going over both of these limits, one temporary workaround is to create a new site just to store certain large assets like images, PDFs, or slide decks. For example, the [Federal Retirement Thrift Savings Board](https://www.frtib.gov/) (FRTIB) hosts all their archived meeting minutes from another Github repo which deploys to https://minutes.frtib.gov and links to these from the primary site.
```
Comment (apburnes) [inline pages/documentation/large-file-handling.md:16]: Any mention of an agency needs approval via an IAA so let's keep it general. We can work with our business team to put together something but that will take time.

**Review by apburnes — COMMENTED** [review]
We don't have any example tools for compression or image optimization but highlighting the strategy is a decent start.

**Review by apburnes — APPROVED** [review]
Thanks! :shipit:

**Pull request merged**